code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -461,6 +461,53 @@
 		SSticker.mode_result = "loss - rev heads killed"
 		SSticker.news_report = REVS_LOSE
 
+//////////////////////////////////////////////
+//                                          //
+//                 FAMILIES                 //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/roundstart/families
+	name = "Families"
+	persistent = TRUE
+	antag_flag = ROLE_FAMILIES
+	restricted_roles = list("Cyborg", "AI", "Prisoner","Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Head of Personnel")
+	required_candidates = 2 // will always have at least 1 family + opponent for that family (other family or cop)
+	weight = 2
+	cost = 30
+	requirements = list(101,101,101,80,70,30,15,10,10,10)
+	high_population_requirement = 10
+	flags = TRAITOR_RULESET
+	blocking_rules = list(/datum/dynamic_ruleset/midround/families)
```

Comment:
You got the blocking_rules the wrong way, this should go the midround ruleset with 
`/datum/dynamic_ruleset/roundstart/families`.

Reply:
that line is already there, i was just copying what the revs rulesets did.

Reply:
well, this just stops the roundstart ruleset from being triggered AFTER the midround has been triggered.